### PR TITLE
Updated to Ant Design v4.19.1

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
N/A

## Related issue
N/A

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [X] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
~~* [ ] Tests added (or description of how to test) for any new features.~~
~~* [ ] User documentation updated for UI or technical changes.~~

### CODE REVIEW

File: src/main/webapp/resources/js/components/Header/MainNavigation.jsx [hunk 1]
```diff
@@ -1,227 +1,202 @@
...
-    <Menu.Item key="userguide">
-      <a
-        href="https://irida.corefacility.ca/documentation/user/user"
-        target="_blank"
-        rel="noreferrer"
+export function MainNavigation() {
+  return (
+    <StyledHeader>
+      <Menu
+        theme="dark"
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/components/Header/MainNavigation.jsx:50]: Change this `theme="dark"` to use the `theme` from `theme-utilities.js`

File: src/main/webapp/resources/js/components/Header/main-navigation/components/AnnouncementsSubMenu.jsx [hunk 2]
```diff
@@ -48,8 +47,24 @@ export function AnnouncementsSubMenu() {
     });
   }
 
-  const aMenu = (
-    <Menu className="t-announcements-submenu" theme={theme}>
+  return (
+    <Menu.SubMenu
+      theme="light"
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/components/Header/main-navigation/components/AnnouncementsSubMenu.jsx:52]: Same as above, let's use the theme from theme-utilities

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
Almost there. Just the few issues we discussed with the resizing of the screen and the few comments below

File: src/main/webapp/resources/js/components/Header/MainNavigation.jsx (hunk 1 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/components/Header/MainNavigation.jsx]: Updated

File: src/main/webapp/resources/js/components/Header/main-navigation/components/AnnouncementsSubMenu.jsx (hunk 2 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/components/Header/main-navigation/components/AnnouncementsSubMenu.jsx]: Updated

File: src/main/webapp/resources/js/components/Header/MainNavigation.jsx [hunk 3]
```diff
@@ -1,227 +1,264 @@
...
+          theme={theme}
+          mode="horizontal"
+          style={{
+            display: "inline-block",
+            width: 100,
+          }}
+        >
+          <Menu.SubMenu key="small" title={i18n("nav.main.small")}>
+            <Menu.SubMenu key="projects" title={i18n("nav.main.projects")}>
+              <Menu.Item key="project:list">
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/components/Header/MainNavigation.jsx:139]: href links missing for `your projects` and `list all projects`

File: src/main/webapp/resources/js/components/Header/main-navigation/components/CartLink.jsx [hunk 4]
```diff
@@ -38,14 +36,13 @@ export function CartLink() {
...
-      <Button
-        type="link"
-        className="t-cart-count"
-        href={setBaseUrl(`/cart/${inGalaxy ? "galaxy" : "pipelines"}`)}
-        icon={<IconShoppingCart data-count={count} />}
-        style={{ color: textColor }}
-      />
-    </Badge>
+    <a
+      style={{ padding: `0 ${SPACE_MD}` }}
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/components/Header/main-navigation/components/CartLink.jsx:40]: I'm assuming this is what is causing the cart link to display in a weird position on certain screen sizes when resizing?

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
All the pages appear to still be working as expected. Only a few things below all related to the main navigation

File: src/main/webapp/resources/js/components/Header/main-navigation/components/CartLink.jsx (hunk 4 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/components/Header/main-navigation/components/CartLink.jsx:40]: Fixed

File: src/main/webapp/resources/js/components/Header/MainNavigation.jsx [hunk 5]
```diff
@@ -1,227 +1,264 @@
...
+      ) : (
+        <Menu
+          theme={theme}
+          mode="horizontal"
+          style={{
+            display: "inline-block",
+            width: 100,
+          }}
+        >
+          <Menu.SubMenu key="small" title={i18n("nav.main.small")}>
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/components/Header/MainNavigation.jsx:137]: Removed

**Review by deepsidhu85 — APPROVED** [review]

**Pull request merged**